include/device/map.h

```diff
@@ -18,6 +18,13 @@
 
 #include <cpu/difftest.h>
 
+typedef enum {
+  SKIP_FREE = 0,
+  MMIO_READ = 1,
+  MMIO_WRITE = 2,
+  MMIO_EXEC = 4
+} SKIP_REF_FLAG;
```

Comment:
Is it approriate to call this enum "SKIP_REF_FLAG"?

I think it's mmio_diff_type?

Reply:
aleady fixed

Reply:
wip